packages/spaces/commands/wait.js

```diff
@@ -30,7 +30,12 @@ async function run(context, heroku) {
     space = await heroku.get(`/spaces/${spaceName}`, {headers})
   }
 
-  space.outbound_ips = await heroku.get(`/spaces/${spaceName}/nat`)
+  try {
+    space.outbound_ips = await heroku.get(`/spaces/${spaceName}/nat`)
+  } catch (error) {
+    cli.warn(`Retrieving NAT details for the space failed with ${error}`)
```

Comment:
Under what condition do we expect this error message to be relevant? I suspect it's not all possible errors.

Reply:
This API call , essentially shows the Outbound IP's, and I think if that API fails , we can just show all other relevant information. Interestingly, this information (cli.warn) is not shown  to output unlike the info.